src/momento/logs.py

```diff
@@ -25,9 +23,9 @@ def add_logging_level(level_name: str, level_num: int, method_name: Optional[str
 
     To avoid accidental clobberings of existing attributes, this method will
     raise an `AttributeError` if the level name is already an attribute of the
-    `logging` module or if the method name is already present
+    `logging` module or if the method name is already present.
 
-    Example
+    Example::
```

Comment:
Make sure this renders properly without a blank line after. I had some trouble with example code not respecting newlines.

Reply:
Looks good in VS Code.

Reply:
Looks good in VS Code. I think the (pre-existing) dashes took care of that